src/main/scala/spray/json/JsValue.scala

```diff
@@ -28,6 +28,7 @@ sealed abstract class JsValue {
   def toString(printer: (JsValue => String)) = printer(this)
   def compactPrint = CompactPrinter(this)
   def prettyPrint = PrettyPrinter(this)
+  def sortedPrint = SortedPrinter(this)
```

Comment:
this should be fine, no one is supposed to extend this class other than spray-json itself